docs/source/docker/getting_started/selection.rst

```diff
@@ -0,0 +1,533 @@
+- The strategy itself defines the objective to apply on the input data.
+
+Lightly allows you to specify several objectives at the same time. The algorithms try to fulfil all objectives simultaneously.
+
+Lightly's data selection algorithms are supporting three types of input:
+
+- **Embeddings** (either automatically computed using our OSS framework or provided by your model)
+- (Optional)  :ref:`Model predictions <docker-datasource-predictions>` such as classifications, object detections or segmentations
+- (Optional) :ref:`Custom metadata <docker-datasource-metadata>` can be anything you can encode in a json file (from numbers to categorical strings)
+
+.. warning:: Using the selection config is a new feature and **selecting a 
+             proportion of samples (e.g. 50%) is not supported at the moment**, 
+             but will be added soon.
+
+
+
+Prerequisites
+-------------
+
+For using the selection, you must have
+
+- Started the Lightly Worker in worker mode. See :ref:`worker-register`.
+
+- Set up a dataset in the Lightly Platform with a cloud storage as datasource. See :ref:`worker-creating-a-dataset`
+
+Scheduling a Lightly Worker run with selection
+----------------------------------------------
+
+For scheduling a Lightly Worker run with a specific selection,
+you can use the python client and its :py:meth:`lightly.api.ApiWorkflowClient.schedule_compute_worker_run` method.
+You specify the selection with the :code:`selection_config` argument.
+
+Here is an example for scheduling a Lightly worker run with a specific selection configuration:
+
+.. literalinclude:: ../integration/examples/trigger_job.py
+
+
+
+Selection Configuration
+-----------------------
+
+The configuration of a selection needs to specify both the maximum number of samples to select and the strategies:
+
+.. code-block:: python
+
+    {
+        "n_samples": 50,
+        "strategies": [
+            {
+                "input": {
+                    "type": ...
+                },
+                "strategy": {
+                    "type": ...
+                }
+            },
+            ... more strategies
+        ]
+    }
+
+The variable :code:`n_samples` must be a positive integer specifying the absolute number of samples which should be selected.
+
+Each strategy is specified by a :code:`dictionary`, which is always made up of an :code:`input` and the actual :code:`strategy`.
+
+.. code-block:: python
+
+    {
+        "input": ...,
+        "strategy": ...
+    },
+
+
+Selection Input
+^^^^^^^^^^^^^^^^
+
+The input can be one of the following:
+
+.. tabs::
+
+    .. tab:: EMBEDDINGS
+
+        If you don't provide your own
+        embeddings, the `lightly OSS framework for self supervised learning <https://github.com/lightly-ai/lightly>`_ is used. It generates 32-dimensional
+        embeddings with default settings using self-supervised learning. 
```

Comment:
I would remove the part about "32-dimensional embeddings" and "default settings". I would only write "lightly framework for self supervised learning is used to generate embeddings.